launcher/control.py

```diff
@@ -248,15 +261,13 @@ def init(self):
             dict({
                 "_id": project["_id"],
                 "icon": DEFAULTS["icon"]["project"],
+                "type": project["type"],
                 "name": project["name"],
             }, **project["data"])
             for project in sorted(io.projects(), key=lambda x: x['name'])
             if project["data"].get("visible", True)  # Discard hidden projects
         ])
 
-        frame = {"environment": {}}
-        self._frames[:] = [frame]
```

Comment:
We need these two lines or `KeyError: environment` will raised later in `on_project_changed` (at line 332).